src/daemon/command_server.cpp

```diff
@@ -364,61 +365,165 @@ t_command_server::t_command_server(
     , "print_sn_state_changes <start_height> [end height]"
     , "Query the state changes between the range, omit the last argument to scan until the current block"
     );
-#if defined(LOKI_ENABLE_INTEGRATION_TEST_HOOKS)
-    m_command_lookup.set_handler(
-      "relay_votes_and_uptime", std::bind([rpc_server](std::vector<std::string> const &args) {
-        rpc_server->on_relay_uptime_and_votes();
-        return true;
-      }, p::_1)
-    , ""
-    );
 
+#if defined(LOKI_DEBUG)
+    char const debug_cmd_fmt[] = "debug [[[toggle|set] <setting>] | [mine_n_blocks <address> <n_blocks>] | [relay_votes_and_uptime]";
     m_command_lookup.set_handler(
-      "integration_test", std::bind([rpc_server](std::vector<std::string> const &args) {
-        bool valid_cmd = false;
-        if (args.size() == 1)
+      "debug", std::bind([rpc_server, debug_cmd_fmt](std::vector<std::string> const &args) {
+
+        LOKI_DEFER
         {
-          valid_cmd = true;
-          if (args[0] == "toggle_checkpoint_quorum")
+          #if defined(LOKI_ENABLE_INTEGRATION_TEST_HOOKS)
+            loki::write_redirected_stdout_to_shared_mem();
+          #else
+            std::cout << std::endl; // NOTE: Makes sure to flush the output stream because monero logging does something funky with buffering
```

Comment:
Is the \n desired here, or would `std::cout << std::flush;` suffice here to flush without appending a newline?